README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is a set of tags that allow developers to better organize and keep track of the elements on their page, and also later be able to easily call them when editing in CSS.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+The Block display will make the element block-level, meaning that it will take up an entire block of space on the page.  Nothing can be sat next to it.  Whereas the Inline property allows elements to sit next to each other in the same line, without pushing other elements around it further down the page.
+
 3. What are the 4 areas of the box model?
 
+Content, padding, margin, and border
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+It works on the cross axis.
+
 5. Explain why git is valuable to a team of developers.
 
+Git allows a developer to pull down a project and work on it without affecting the project or their teammates work by created their own branch.  Once they complete the work on their own branch, it can be pushed back up and reviewed for bugs and errors before it is merged with the main project.
```

Comment:
Great answers, Git is a powerful tool and it seems you have a solid understanding of its functions. As you progress through the lambda school curriculum you will use git daily and become familiar with even more functions and features